app/lib/frontend/templates.dart

```diff
@@ -578,6 +577,5 @@ abstract class LogoUrls {
   static const String flutterLogo32x32 = '/static/img/flutter-logo-32x32.png';
 }
 
-const String flutterPluginsDescriptionHtml =
-    '<p><a href="https://flutter.io/platform-plugins/">'
-    'Learn more about Flutter plugins.</a></p>';
+const String flutterPackagesDescriptionHtml =
+    '<p><a href="https://flutter.io/">Learn more about Flutter.</a></p>';
```

Comment:
@isoos I think this is fine for now

@mit-mit @eseidelGoogle 

At the moment we have a link on the site (see here https://pub.dartlang.org/flutter/plugins) that points folks to flutter/platform-plugins

Any desire to do something different now that the index is covering all flutter packages?